pkg/backend/display/ai.go

```diff
@@ -71,4 +71,15 @@ func RenderCopilotErrorSummary(summary *CopilotErrorSummaryMetadata, err error,
 		fmt.Fprintln(out, "  "+opts.Color.Colorize(colors.BrightGreen+line+colors.Reset))
 	}
 	fmt.Fprintln(out)
+
+	if opts.ShowLinkToCopilot {
+		PrintCopilotLink(out, opts, permalink)
+	}
+}
+
+func PrintCopilotLink(out io.Writer, opts Options, permalink string) {
+	fmt.Fprintln(out, "  "+"Would you like additional help with this update?")
+	fmt.Fprintln(out, "  "+
+		opts.Color.Colorize(colors.Underline+colors.Blue+ExplainFailureLink(permalink)+colors.Reset))
```

Comment:
For our other permalink we seem to be using `BrightBlue`: https://github.com/pulumi/pulumi/blob/master/pkg/backend/display/display.go#L56

We should maybe factor this out into `color.SpecLink` or something, but that should go to a separate PR, so dunno if you want to do it now.

Reply:
Good shout, looks much better w/ my terminal palette too. Will leave the SpecLink for now and make a follow up issue to track.